expertsystem/schemas/xml/particle.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$id": "https://github.com/ComPWA/expertsystem/blob/master/schemas/xml/particle.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#definitions/Particle",
+  "definitions": {
+    "Particle": {
+      "type": "object",
+      "required": ["Name", "Pid", "Parameter", "QuantumNumber"],
+      "additionalProperties": false,
+      "properties": {
+        "Name": { "type": "string" },
+        "Pid": {
+          "anyOf": [
+            { "type": "string", "pattern": "^([+-]?[0-9]+)$" },
+            { "type": "integer" }
+          ]
+        },
+        "Parameter": { "$ref": "#/definitions/Parameter" },
+        "QuantumNumber": { "$ref": "#/definitions/QuantumNumbers" },
+        "DecayInfo": {
+          "type": "object",
+          "required": ["Parameter"],
```

Comment:
You're right. Actually in the next PR this requirement is removed, because it makes the tests fail when checking the `ParticleList` of an output recipe.

Reply:
Ok, since this code is temporary anyway, I leave it up to you if you want to fix it here or in the next PR.

Reply:
08b8f85699384989831319fc84602dd462bfdf9a